packaging/Dockerfile.centos.7

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Please remove Docker files.

Reply:
Actually all of `packaging/` seem to be DAOS specific.

Reply:
Yes. It looks to me this whole PR is to the wrong repo.